### PR TITLE
Add admin functions

### PR DESCRIPTION
**Pull request by gitreadygitset**

### CODE REVIEW

**Comment by gitreadygitset** [comment]
Will add test in next PR

File: app/controllers/api/v1/reviews_controller.rb [hunk 1]
```diff
@@ -1,21 +1,36 @@
...
 
   def review_params
     params.require(:review).permit(:rating, :comment)
   end
+
+  def authorize_user
+    if !user_signed_in? || !(current_user.role == "admin")
+      render json: { error: ["Only admins have access to this feature"] }
+    end
+  end
```
Comment (nwalberts) [inline app/controllers/api/v1/reviews_controller.rb:35]: beautiful. you are beautiful, whoever you are

File: app/javascript/react/components/ReviewsContainer.js [hunk 2]
```diff
@@ -2,9 +2,23 @@ import React from "react";
...
+      }
+
+      return <ReviewTile 
+        key={review.id} 
+        id={review.id} 
+        rating={review.rating} 
+        comment={review.comment} 
+        handleClick={handleClick} 
+        currentUser={props.currentUser}
+      />;
```
Comment (nwalberts) [inline app/javascript/react/components/ReviewsContainer.js]: I get the intention behind this indentation, but we do unfortunately want towards:
- wrapping what is in your return in `()`
- ensuring consistent indentation for the React component invocation

due to the number of props you are aiming to pass 
https://github.com/airbnb/javascript/tree/master/react#alignment

File: app/uploaders/profile_photo_uploader.rb [hunk 3]
```diff
@@ -10,7 +10,7 @@ def store_dir
     "public/testuploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
   end
 
-  def default_url
-    ActionController::Base.helpers.asset_path("fallback/" + [version_name, "https://daily-buzz-profile-pics.s3.amazonaws.com/coffee-791045_1280.jpg"].compact.join('_'))
-  end
+  # def default_url
+  #   ActionController::Base.helpers.asset_path("fallback/" + [version_name, "https://daily-buzz-profile-pics.s3.amazonaws.com/coffee-791045_1280.jpg"].compact.join('_'))
+  # end
```
Comment (nwalberts) [inline app/uploaders/profile_photo_uploader.rb]: if this is commented out for now, I would just add this code to your trello board and remove. I get why you have it, but we do want to keep our PRs clean

**Pull request merged**